wasm-node/javascript/src/public-types.ts

```diff
@@ -177,6 +177,20 @@ export interface Chain {
      */
     nextJsonRpcResponse(): Promise<string>;
 
+    /**
+     * JSON-RPC responses or notifications async iterable.
+     *
+     * Each chain contains a buffer of the responses waiting to be sent out. Iterating over this
+     * pulls one element from the buffer. If the iteration happen at a slower rate than
+     * responses are generated, then the buffer will eventually become full, at which point further iteration on
+     * {@link Chain.jsonRpcResponses} will throw an exception. The size of this buffer can be configured
```

Comment:
It's `sendJsonRpc` that would throw an exception.

Reply:
Oops, thanks for the catch!